config/sudoers

```diff
@@ -21,6 +21,7 @@ root	ALL=(ALL:ALL) ALL
 
 # fine-grained sudo permissions for certain users & actions
 vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/set-clock.sh
+vx-admin ALL=(root:ALL) NOPASSWD: /vx/admin/admin-functions/rekey_via_tpm.sh
```

Comment:
nit: Looks like all other `admin-functions` scripts use kebab-case file names instead of snake_case